manifoldco/ui
# added tooltip to number input

## Reason for change

Added tooltips to number input component. 
(The text content isn't necessarily the best imo, but we can iterate on that.)


![tooltip](https://user-images.githubusercontent.com/10498708/55106338-6c6a9780-50ad-11e9-8ad7-ff405b1bd638.gif)

Original Design: https://projects.invisionapp.com/share/TDR3AFGA7WU#/screens/353097727_Custom

## Testing

- Go to plan selector
- Make sure the tooltip appears when the button is disabled, and doesn't appear when the button is enabled.
